IPython/core/oinspect.py

```diff
@@ -702,15 +702,22 @@ def pinfo(
             del info['text/html']
         page.page(info)
 
-    def info(self, obj, oname="", formatter=None, info=None, detail_level=0):
-        """DEPRECATED since 5.0. Compute a dict with detailed information about an object."""
-        if formatter is not None:
-            warnings.warn('The `formatter` keyword argument to `Inspector.info`'
-                     'is deprecated as of IPython 5.0 and will have no effects.',
-                      DeprecationWarning, stacklevel=2)
-        return self._info(obj, oname=oname, info=info, detail_level=detail_level)
+    def _info(self, obj, oname="", info=None, detail_level=0):
+        """
+        Inspector.info() was likely unproperly marked as deprecated
```

Comment:
```suggestion
        Inspector.info() was likely improperly marked as deprecated
```

"unproperly" is not a word in English, its "improperly" you're looking for.

Yes, English is weird and discontinuous—er, no, unconsistant—er, inconsistent.